build/x86_64_sles15/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM opensuse/leap:15.1
```

Comment:
Just for the records, Opensuse changed their version from 42.X to 15.X for SLES 15.

![image](https://user-images.githubusercontent.com/5287526/66867509-711f6c00-ef69-11e9-8926-55f9f3f538de.png)